tests/SpecTests/command-monitoring/find.json

```diff
@@ -356,7 +356,7 @@
               },
               "collection": "test",
               "batchSize": {
-                "$numberLong": "1"
+                "$numberLong": "3"
```

Comment:
I assume all of the spec test changes are from https://github.com/mongodb/mongo-php-library/pull/828/commits/6d7fd374ca14a80edabeefdd27cdd6187c43a0eb. Should that commit message refer to the commit from the specs repo that you synced with?

Reply:
Updated commit to include the commit hash.

This change is necessary to fix tests on 5.0 regarding cursor behaviour. I've extracted those changes into a separate commit so that it can be tracked (and reverted if necessary) easily.

Reply:
Does this conflict with [PHPLIB-666](https://jira.mongodb.org/browse/PHPLIB-666)?

Reply:
I've removed this particular change since the original issue is now resolved.